CmsWeb/Areas/Org/Views/Org/Registration/RegistrationEdit.cshtml

```
@@ -146,6 +146,12 @@
           </label>
           <label class="checkbox-inline control-label">
             @Html.CheckBoxFor(m => m.NotReqMarital) Marital status not required
+          </label>
+            <label class="checkbox-inline control-label">
+            @Html.CheckBoxFor(m => m.ShowDOBOnFind) Find does not show Day of Birthday
+          </label>
+          <label class="checkbox-inline control-label">
+              @Html.CheckBoxFor(m => m.ShowPhoneOnFind) Find does not show Phone
```

`Find shows Phone Number` - I assume the `ShowPhoneOnFind` field should be `true` by default and toggled off to hide it otherwise this becomes confusing.